firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -43,6 +43,10 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
 
     // Presents alert to clear users private session data
     func tabToolbarDidPressFire(_ tabToolbar: TabToolbarProtocol, button: UIButton) {
+        didTapOnFire()
+    }
+
+    func didTapOnFire() {
```

Comment:
Should we maybe add a comment to the method or rename it? Tapping on fire is not very self explanatory unless you have knowledge of the feature itself.